app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    input_command, file, new_file = command.split()
```

Comment:
Validate before that command have exactly 3 parameters
Check code for this cases:
mv file1.txt
mv file1.txt file2.txt file3.txt